deploy/firesim

```diff
@@ -240,6 +257,8 @@ def construct_firesim_argparser():
                         default=-1)
     parser.add_argument('-q', '--forceterminate', action='store_true',
                         help='For terminaterunfarm, force termination without prompting user for confirmation. Defaults to False')
+    parser.add_argument('-t', '--launchtime', type=str,
+                        help='Give the "Y-m-d--H-M-S" prefix of results-build directory. Useful for tar2afi when finishing a partial buildafi')
```

Comment:
I currently only added support for `--launchtime` to the GlobalBuildConfig.  It will also be useful for the RuntimeConfig if we refactor the monitor loop in `runworkload` into a separate method that can be restarted if/when the monitor dies.